spec/javascripts/stimulus/tabs_and_panels_controller_test.js

```diff
@@ -2,16 +2,26 @@
  * @jest-environment jsdom
  */
 
-import { Application } from "stimulus";
-import tabs_and_panels_controller from "../../../app/webpacker/controllers/tabs_and_panels_controller";
+import { Application } from 'stimulus';
+import tabs_and_panels_controller from '../../../app/webpacker/controllers/tabs_and_panels_controller';
```

Comment:
Why single quotes? I don't think that we have a convention here. Maybe we need one... I personally prefer double quotes. :wink:

Reply:
Old habits, I tend to prefer single quotes :smile:  It probably comes from my PHP days.